admin-frontend/src/utils.js

```diff
@@ -4,7 +4,13 @@ import { dataProvider } from './providers/dataProvider';
 export const genderChoices = [
```

Comment:
Minor comment: These could be extracted to constants etc. file, where some constant data or variables are stored. Imo utils are generally utility functions. But doesn't really matter for now if it works.

Reply:
yeah, that's how it was before me, I didn't even think to reorganise the structure...

Reply:
Yeah I kind guessed that might be the case. Well then it's not your responsibility to fix it only, so just leave it like that :)